pkg/apis/componentconfig/fuzzer/fuzzer.go

```diff
@@ -1,29 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package fuzzer
-
-import (
-	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
-)
-
-// Funcs returns the fuzzer functions for the componentconfig api group.
-// TODO: When the componentconfig types are split out to individual API groups
```

Comment:
do we have an issue for this?

Reply:
not yet. let's create asap

Reply:
Created https://github.com/kubernetes/kubernetes/issues/68244